### ISSUE TITLE
Contents and Trickle

### ISSUE THREAD

**Issue by simondate** [KingsOnline/adapt-contents#1]
Been testing Contents with the Trickle extension and it seems that the page shows as locked. Then when you click the elements which are below Trickle you are navigated to the top of the page

**Comment by robmausser** [comment]
Yes Trickle doesnt work and also it does not show that the specific component is inaccessible like it does in the Progress Menu (greyed out)

**Comment by robertmausser** [comment]
It also doesnt autoscroll with trickle (you arent brought to the next component)
